cumulusci/tasks/metadata_etl/layouts.py

```diff
@@ -68,6 +69,262 @@ def _create_new_related_list(self, metadata, api_name, related_list):
         elem.append("relatedList", text=related_list)
 
 
+class AddFieldsToPageLayout(MetadataSingleEntityTransformTask):
+    task_docs = """
+        Inserts the listed fields or Visualforce pages into page layouts
+        specified by API name.
+
+        If the targeted item already exists, the layout metadata is not modified.
+
+        Fields property options:
```

Comment:
Should "fields" be singular here?
```suggestion
        Field property options:
```